deploy/PortabilityService.ServiceFabricApplication/DeploySF.ps1

```diff
@@ -131,6 +137,10 @@ if ($EnvironmentName)
 {
     $parameters.EnvironmentName = $EnvironmentName
 }
+if ($ImageTag)
+{
+    $parameters.ImageTag = $ImageTag
```

Comment:
What's the point of this if the tags cannot be parameterized?

Reply:
Just added for future use. Though researching, there may not be an easy way to use it so I'll remove it for now.